examples/non-monorepo/yarn.lock

```diff
@@ -1,1815 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Was this needed?

Reply:
Probably not although it's not necessary either way